web/docs/02_how_to/1.GameTranslation.stories.mdx

```diff
@@ -14,7 +14,10 @@ The following table describe all languages acceptable:
 
 | Language | Description        |
 | -------- | ------------------ |
-| en       | English            |
+| en       | English (Default)  |
```

Comment:
Should we set English (Description) or each language should have in each language?

Reply:
I think it is ok to expect developers to use English